plugins/thunderbolt/fu-self-test.c

```diff
@@ -779,22 +779,22 @@ static MockDevice root_one = {
 
     .name = "Laptop",
     .id = "0x23",
-    .nvm_version = "20.2",
-    .nvm_parsed_version = "20.02",
+    .nvm_version = "20.02",
```

Comment:
is this right?  IIRC `nvm_version` should reflect real files on the system and `nvm_parsed_version` should be how fwupd parsed it.